templates/index

```diff
@@ -145,6 +145,7 @@ cisco_ios_show_version.template, .*, cisco_ios, sh[[ow]] ver[[sion]]
 cisco_ios_show_ip_arp.template, .*, cisco_ios, sh[[ow]] i[[p]] a[[rp]]
 cisco_ios_show_ip_bgp.template, .*, cisco_ios, sh[[ow]] i[[p]] bgp
 cisco_ios_show_clock.template, .*, cisco_ios, sh[[ow]] clo[[ck]]
+cisco_ios_show_tacacs.template, .*, cisco_ios, sh[[ow]] tacacs
```

Comment:
This should be moved above the previous entry as the command is one character longer and is the least preferred alphabetically from commands of the same length.

Reply:
Agreed on this